### PR TITLE
docs: import demo code

### PR DESCRIPTION
**Pull request by andretchen0**
Problem: on-screen example code sometimes doesn't work – e.g., [it includes typos, like this.](https://github.com/Tresjs/tres/pull/411)

Solution (partial): import the demo source code for display, rather than copy/pasting. At least that way, the demo and on-screen source code remain synced.

Background: see ["Importing Code Snippets" in the Vitepress docs.](https://vitepress.dev/guide/markdown#import-code-snippets)

### CODE REVIEW

File: docs/.vitepress/theme/components/SVGDemo.vue [hunk 1]
```diff
@@ -2,7 +2,8 @@
 import { TresCanvas } from '@tresjs/core'
 import { OrbitControls, SVG } from '@tresjs/cientos'
 
-const svgURL = 'https://raw.githubusercontent.com/Tresjs/assets/main/svgs/cientos_heart.svg'
+const svgURL = 'https://raw.githubusercontent.com/'
```
Comment (JaimeTorrealba) [inline docs/.vitepress/theme/components/SVGDemo.vue:5]: Just asking, why this change?

File: docs/guide/loaders/svg.md [hunk 2]
```diff
@@ -8,29 +8,7 @@ A wrapper around the `three` [SVGLoader](https://threejs.org/examples/?q=sv#webg
...
-        :src="svgURL"
-        :position="[-0.4, 1, 0]"
-        :scale="0.01"
-      />
-    </Suspense>
-    <TresGridHelper />
-  </TresCanvas>
-</template>
-```
+<<< @/.vitepress/theme/components/SVGDemo.vue{3,12-18}
```
Comment (JaimeTorrealba) [inline docs/guide/loaders/svg.md:11]: I really like this Idea, but I would encourage also for standardization. Would you like to create a new issue to use this way for all demos?
idk something like "migration docs" or so?

**Review by JaimeTorrealba — APPROVED** [review]

File: docs/.vitepress/theme/components/SVGDemo.vue (hunk 1 above)
Comment (andretchen0) [reply docs/.vitepress/theme/components/SVGDemo.vue:5]: It's just so that the demo code fits on screen without a horizontal scrollbar.

File: docs/guide/loaders/svg.md (hunk 2 above)
Comment (andretchen0) [reply docs/guide/loaders/svg.md:11]: Sure. I'll open an issue.
